plugins/generator-1.15.2/forge-1.15.2/mappings/types.yaml

```diff
@@ -7,4 +7,5 @@ map: HashMap
 string: String
 boolean: boolean
 direction: Direction
-advancement: Advancement
\ No newline at end of file
+advancement: Advancement
+projectileentity: Entity
```

Comment:
This mapping is probably no longer needed with new immediatesourceentity

Reply:
Yeah you're right, sorry I forgot it while removing the mappings. It's now fixed.